pagination/ResidentialProperties/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Container, LoadingOverlay, Paper, SimpleGrid, Title } from "@mantine/core";
+import Image, { StaticImageData } from "next/image";
+import React, { useEffect, useState } from "react";
+import { AppCard } from "../../components";
+import { useAuth } from "../../hooks/AuthContext";
+import { residential_properties_data } from "../../pseudodata";
+import style from "./_index.module.scss";
+
+type ResidentialProperty = {
+  description: string;
+  imageSrc: StaticImageData;
+  title: string;
+  type: string;
+  values: {
+      amount: string;
+      area: string;
+      car: string;
+      people: string;
+  };
+}
+
+function ResidentialProperties() {
+  const { user } = useAuth();
+  const [residentialProperties, setResidentialProperties ] = useState<ResidentialProperty[]>();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    async function getResidentialProperties() {
+      setIsLoading(true);
+      const data = residential_properties_data;
+
+      if (data) {
+        setResidentialProperties(data);
+      }
+      
+      setTimeout(()=>setIsLoading(false), 1000);
```

Comment:
Don't mind this lng haha iremove raning setTimeout if naa njud tong await fetching sa data or idk unsay way ani sa kang jez hahaha para rni mu loading